### PR TITLE
Notes for the first section in week 8 lecture

### PR DESCRIPTION
**Pull request by Florence2Me**
In this section, we focused on the introduction of contrastive methods in Energy-Based Models in several aspects. First, we discuss the advantage brought by applying contrastive methods in self-supervised learning. Second, we discussed the architecture of denoising autoencoders and their weakness in image reconstruction tasks. We also talked about other contrastive methods, like contrastive divergence and persistent contrastive divergence.

### CODE REVIEW

File: docs/en/week08/08-1.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+---
+lang-ref: ch.08-1
+title:
```
Comment (Atcold) [inline docs/en/week08/08-1.md]:
```suggestion
title: Energy-based Models, Part II
```

File: docs/en/week08/08-1.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
+---
+lang-ref: ch.08-1
+title: 
+authors: Vishwaesh Rajiv, Wenjun Qu, Xulai Jiang, Shuya Zhao
+date: 23 Mar 2020
+---
+
+# Energy-based Models, Part II
```
Comment (Atcold) [inline docs/en/week08/08-1.md]:
```suggestion
```

**Comment by Atcold** [comment]
There is no summary in `08.md`.
Also, there is no information whatsoever in the spreadsheet.

**Comment by Florence2Me** [comment]
I'll add a shorter summary to 08.md.

Best,
Shuya

**Comment by Atcold** [comment]
Why did you close it?

**Comment by Atcold** [comment]
- Replace figure 4 with actual math.
 - Set `width="50%"` for fig 1 and 2.
 - `neg` should be `\text{neg}`, otherwise it looks like the product of `n`, `e`, and `g`.
 - I'm not sure what's that non math `F` in the picture below.

Screenshot 2020-03-28 20 35 31

**Pull request merged**